### DISCUSSION TITLE
Figma to Webstudio to Github

### DISCUSSION THREAD

**Discussion by JoshuaMadrid** [webstudio-is/webstudio-community#188] [Ideas]
I love to see something like a push to webstudio from figma in components or blocks of content, maybe the same copy and paste feature of webflow. Then a way to export to github maybe thats a pro feature? with that i can then have other developers look at the new branch created by webstudio and check for important conflicts or make sure additions that were added outside of webstudio are unaffected by the merge.

**Comment by kof** [comment]
With the CLI you can generate code and create a pull request https://docs.webstudio.is/university/self-hosting

But since there are no user-defined components https://github.com/orgs/webstudio-is/projects/11?pane=issue&itemId=76022909 yet its not going to be very practical to import that code directly from code-driven code-base if that's what you are thinking.

This workflow will be well supported once we have user-defined components.
